lessons/AI/ai-statistical-lang-models/langs/en-us/pages/soekia-closer-look.adoc

```diff
@@ -0,0 +1,64 @@
+= Soekia: A Closer Look
+
+Go to @link{http://Soekia.ch/GPT/?lang=en}. Click "Look Inside." Access the "Weather" collection from the green "Documents" panel, then respond to the prompts below.
+
+== Look Inside: N-grams
+
+@n In the yellow n-grams panel, click the red vertical "Create n-Grams" button. After you click, you will see a red circle appear on each document, one at a time and in sequence. What do you think this red circle represents? @fitb{}{}
+
+@fitb{}{}
+
+@n Set the N-grams to unigrams by clicking "1er". Click the "Create n-grams" button. You should see all of the N-grams populate.
+
+@n Locate the n-gram with the longest green line. Click on it, then use the information that appears to complete the first column of the table below. (*Don't panic* if your answers are different from your peers'! That's exactly what _should_ happen.)
+
+@vspace{1ex}
+
+[cols="<.^2,^.^1,^.^1,^.^1", stripes="none", options="header"]
+|===
+
+|
+| 1-Gram
+| 3-gram
+| 5-gram
+
+| What is the N-gram? |||
+| How many times was it found? |||
```

Comment:
the answers on my screen are in German. is there a way for kids to see an English version?